examples/config-allextensions.json

```diff
@@ -1,5 +1,5 @@
 {
-	"version": "0.1.0-dev",
+	"distSpecVersion": "0.1.0-dev",
```

Comment:
make the value "1.0.0", since it is now official.

can you point the value to this somehow?
https://github.com/opencontainers/distribution-spec/blob/v1.0/specs-go/version.go#L32

Reply:
Also, change it in all the config sections as well in the examples.

Reply:
Just make sure the go.mod also reflects this version correctly.

https://github.com/project-zot/zot/blob/main/go.mod#L36

Reply:
Should I consider rewriting the config.json file or just warn that the version differs from the one currently imported and correct the config.DistSpecVersion value?

Reply:
rewrite them